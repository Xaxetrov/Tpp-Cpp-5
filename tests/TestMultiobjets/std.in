S s1 -5 5 -15 -10
PC pc1 0 0 0 10 20 20
PC pc2 0 0 0 8 20 20 1 0
R r1 3 3 6 6
OR or1 pc1 pc2 r1
OI oi1 pc1 pc2 r1
OR or1 pc1 pc2 r1
OI oi1 pc1 pc2 r1
OR or2
OI oi2
OR or2 pc1 pc1
OI oi2 r1 r1
OR or2 pc1 pc2 pc3
OI oi2 pc1 pc2 pc3
EXIT
