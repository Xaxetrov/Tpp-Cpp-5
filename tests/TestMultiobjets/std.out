OK
OK
OK
OK
OK
OK
ERR
#Incorrect name
ERR
#Incorrect name
ERR
#Incorrect number of objects
ERR
#Incorrect number of objects
ERR
#Several uses of the same objects
ERR
#Several uses of the same objects
ERR
#Inclusion of a non-existant object
ERR
#Inclusion of a non-existant object

